src/poc/miner_poc_mgr.erl

```diff
@@ -544,8 +544,8 @@ initialize_poc(BlockHash, POCStartHeight, Keys, Vars, Ledger, #state{pub_key = C
                         start_height = POCStartHeight
                     },
                     ok = write_local_poc(LocalPOC, State),
-                    lager:info("started poc for challengeraddr ~p, target: ~p, onionhash ~p",
-                        [?TO_ANIMAL_NAME(Challenger), OnionKeyHash]),
+                    lager:info("started poc for challenger addr ~p, target: ~p, onionhash ~p",
+                        [?TO_ANIMAL_NAME(Challenger), libp2p_crypto:bin_to_b58(TargetPubkeybin), OnionKeyHash]),
```

Comment:
lets go with the animal name, keep it consistent

Reply:
makes sense.